collector/pkg/component/analyzer/network/network_analyzer.go

```diff
@@ -525,6 +525,14 @@ func (na *NetworkAnalyzer) getConnectFailRecords(mps *messagePairs) []*model.Dat
 
 func (na *NetworkAnalyzer) getRecords(mps *messagePairs, protocol string, attributes *model.AttributeMap) []*model.DataGroup {
 	evt := mps.requests.event
+	if attributes.HasAttribute(constlabels.HttpContinue) {
```

Comment:
This is a specific case for the HTTP protocol. I think it isn't obvious why we put it here if there are no comments. 
```suggestion
        // See the issue https://github.com/KindlingProject/kindling/issues/388 for details.
	if attributes.HasAttribute(constlabels.HttpContinue) {
```